client/docs/swagger-ui/swagger.yaml

```diff
@@ -29515,6 +29515,881 @@ paths:
                       format: byte
       tags:
         - Query
+  '/panacea/aol/v2/owners/{owner_address}/topics':
```

Comment:
Is auto generation?

Reply:
Yes. Everything in the `client/docs/` are auto-generated by [make proto-swagger-gen](https://github.com/medibloc/panacea-core/blob/83b3983ba35b7be47d296402ed29fdd3fb0a2269/Makefile#L51). That command invokes the [protoc-swagger-gen.sh](https://github.com/medibloc/panacea-core/blob/83b3983ba35b7be47d296402ed29fdd3fb0a2269/scripts/protoc-swagger-gen.sh#L28) script.

That script has two big steps:
1. Generating Swagger docs from all proto files
2. Combining them into one Swagger doc (using `swagger-combine`)

Reply:
OK. Thank you!